modules/token/logic/tokenCacheLogic.js

```diff
@@ -29,4 +29,29 @@ module.exports = class TokenCacheLogic {
       return acc;
     }, Promise.resolve({})));
   }
+
+  static async wordRegistry(req, res) {
+    return res.send(await CacheLogic.getWordRegistryData());
+  }
+
+  static async wordSaleDetails(req, res) {
+    if (!req.query.address) return res.status(400).send('address query missing');
+
+    return res.send(await CacheLogic.wordSaleDetails(req.query.address));
+  }
+
+  static async wordSaleDetailsByToken(req, res) {
+    if (!req.query.address) return res.status(400).send('address query missing');
+
+    const data = await CacheLogic.wordSaleDetailsByToken(req.query.address);
+    if (!data) return res.status(404).send('no word sale information for address');
+
+    return res.send(data);
+  }
+
+  static async wordSaleVotesDetails(req, res) {
+    if (!req.query.address) return res.status(400).send('address query missing');
+
+    return res.send(await CacheLogic.wordSaleVotesDetails(req.query.address));
+  }
```

Comment:
move this to the routes file?

Reply:
we only started this refactoring in the other branch thats not yet in develop, should I already include it here?

Reply:
lets add it as a todo :)